src/key-value-pairs/styles.scss

```diff
@@ -0,0 +1,53 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+@use '../internal/styles' as styles;
+@use '../internal/styles/tokens' as awsui;
+
+.key-value-pairs {
+  @include styles.styles-reset;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+  row-gap: awsui.$space-scaled-m;
+}
+
+.list {
+  @include styles.styles-reset;
```

Comment:
One style reset on the root was enough, wasn't it?

Reply:
Not really, root style reset does not reset some specific `dl` styles

Reply:
Correct, because the point of `styles-reset` is to cancel all inheritable CSS (colors, font-sizes, etc), not set all possible values to default.

You need to reset browser defaults on your own.

Reply:
updated